### PR TITLE
[WIP] Basic logging with Syslog in gregord

### PR DESCRIPTION
**Pull request by jacobhaven**
Very basic logging to help debug gregord issues.
@oconnor663

### CODE REVIEW

**Comment by jzila** [comment]
it seems that you committed the `gregord` binary.

File: gregord/main.go [hunk 1]
```diff
@@ -1,33 +1,52 @@
...
 	"github.com/keybase/gregor/protocol/gregor1"
 	grpc "github.com/keybase/gregor/rpc"
+	"golang.org/x/net/context"
 )
 
+var logger *logging.Logger
+
+func init() {
+	logger = logging.MustGetLogger("gregord")
+	syslogIP, syslogPort := os.Getenv("LOGGLY_PORT_514_UDP_ADDR"), os.Getenv("LOGGLY_PORT_514_UDP_PORT")
```
Comment (jzila) [inline gregord/main.go]: so if these aren't set, we just don't log, correct?

**Comment by oconnor663** [comment]
Just put up https://github.com/keybase/gregor/pull/48

**Comment by oconnor663** [comment]
@jzila this is just a demo PR

File: gregord/main.go (hunk 1 above)
Comment (jacobhaven) [inline gregord/main.go]: No, you should log to stdout/stderr.

**Pull request closed without merging**